enterprise/api_client/lms.py

```diff
@@ -18,7 +18,11 @@
 from django.utils import timezone
 
 from enterprise.constants import COURSE_MODE_SORT_ORDER, EXCLUDED_COURSE_MODES
-from enterprise.utils import NotConnectedToOpenEdX, get_enterprise_worker_user
+try:
+    from enterprise.utils import NotConnectedToOpenEdX, get_enterprise_worker_user
+except ImportError:
+    NotConnectedToOpenEdX = None
+    get_enterprise_worker_user = None
```

Comment:
So adding a `try` around these import statements fixed the initialization error that I was getting when importing EnterpriseCatalogApiClient in `enterprise.utils.py`

But then running `test_forms.py` has errors with this when initializing the mock catalog info.

Reply:
could you remove this change as well as the installed apps added in `enterprise/settings/test.py`?  i would like to see what test failures you got before these changes... i suspect we should be able to resolve by mocking something in a test

Reply:
sure I'll do that

Reply:
Removed those changes